config/drupal-8/drupal-8.8-deprecations.yml

```diff
@@ -0,0 +1,6 @@
+# Contains automatic fixes for some deprecations introduced in Drupal 8.8.
+services:
+  DrupalRector\Rector\Deprecation\PathAliasManagerServiceNameRector: ~
+  DrupalRector\Rector\Deprecation\PathAliasWhitelistServiceNameRector: ~
+  DrupalRector\Rector\Deprecation\PathSubscriberServiceNameRector: ~
+  DrupalRector\Rector\Deprecation\PathProcessorAliasServiceNameRector: ~
```

Comment:
We are creating one configuration line per rector rule and each rule changes one thing if possible. There are other ways to go about this, but we thought this would be a simple way to start.

Reply:
I started with one Rector per rule but found it to be repetitive: it takes more time and effort to write, and I didn't immediately see the advantage.  I was optimizing for speed and simplicity, rather than abstraction.  One Rector per rule certainly works, and might show its advantages in time. 👍

Reply:
Thanks for the feedback!